cve_bin_tool/checkers/wget.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""
+CVE checker for Wget
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-332/GNU-Wget.html
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """
+    returns version information for wget found in given file.
+    VPkg: gnu, wget
+    """
+    regex = [r"Wget: ([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
This is capitalized, and the string here is not:

https://github.com/intel/cve-bin-tool/pull/427/files#diff-e065361581319fab099d11b659374c3cR8

Which is why your tests are failing.

Reply:
Thanks for the reply, I will make the required changes.